client/src/Movies/MovieCard.js

```diff
@@ -1,48 +1,27 @@
 import React from 'react';
-import MovieDetails from './MovieDetails';
+import './movies.css';
+
 
 const MovieCard = (props) => {
-  
+    
   let id = props.match.params.id;
 
   if (!props.movies[id]) {
     return <div>Loading movie information...</div>
   }
 
-  
-  // console.log(props);
-  console.log(id);
-  console.log(props.movies[id].title);
-  
-  // console.log(props.movies[id].title);
-
-        return <div>Movie: {props.movies[id].title} </div> 
-    // return <div>Movie: {props.movies[id].title} </div>;
-  }
+    return (
+      <div className='movie-card'>
+          <button onClick={ () => {return props.addSave(props.movies[id])}}>Add to Your Saved Movies</button>
```

Comment:
Awesome job getting the save functionality working! Note that the return statement here is optional, since react doesn't do anything with the return values